index.html

```diff
@@ -17,7 +17,7 @@
         <blockquote>
```

Comment:
nice work Matt. the code is easy to read and well structured. please checkout my comments below.

after you have implemented the external url, maybe we could go one step further and define a [class](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes) for the functionality. This will not be particularly useful for this relatively small example, but it may serve as a good intro for later exercises/frameworks.

Reply:
Hey Tom,

I've got the external API working, but I'm not sure if I want to actually keep this functionality. The main problem is that the API doesn't work like it's supposed to because the quote data contains a lot of errors. Apart from the quotes containing many spelling and capitalization mistakes, the "censored" feature which allows you to fetch only quotes that do not contain "adult language" doesn't work properly because the author hasn't correctly identified which quotes actually contain adult language: the API censors quotes that do not contain adult language, and fetches ones that do.

Basically I would find it embarrassing/inappropriate to have some of those quotes show up in my app, if you know what I mean.

Should I finish the API version so that you can review it, and then change the code afterwards?

Reply:
ah that is annoying. I am sorry to hear.
let me have a look at it if you don't mind. after all we are doing this to learn. so let's use the chance.